recipes-core/initrdscripts/files/init-readonly-rootfs-overlay-boot.sh

```diff
@@ -61,6 +61,10 @@ read_args() {
 				ROOT_RWRESET=$optarg ;;
 			rootrwoptions=*)
 				ROOT_RWMOUNTOPTIONS_DEVICE="$optarg" ;;
+			overlaymodule=*)
+				OVERLAY_MODULE="$optarg"
+				modprobe $optarg 2> /dev/null || \
```

Comment:
Could you please use "$optargs" (quotes).
I know it is not consistent right now, but we are facing an upcoming patch that will fix such style issues.